slices/gawk.yaml

```diff
@@ -0,0 +1,19 @@
+package: gawk
+
+slices:
+  libs:
+    essential:
+      - libmpfr6_libs
+      - libreadline8_libs
+      - libsigsegv2_libs
```

Comment:
There should be `libc6_libs` and `libgmp10_libs` as well since some libs depend on them.

Reply:
Oh I thought only direct dependencies were required - does this mean that `libmpfr6_libs` won't pull in `libc6_libs` and `libgmp10_libs`, even though they are specified as essential dependencies?

```
package: libmpfr6

slices:
  libs:
    essential:
      - libc6_libs
      - libgmp10_libs
    contents:
      /usr/lib/*-linux-*/libmpfr.so.6*:
```

Reply:
Yup, we only list the direct dependencies even when some slices may come with some of the dependencies inside them. In this case, as per https://packages.ubuntu.com/jammy/gawk, libc6 and libgmp10 are dependencies as well, so it's best to list them here too for better maintainability.